src/main/java/com/gachigang/ontherun/common/utils/InitializationData.java

```diff
@@ -29,5 +34,21 @@ public void run(String... args) {
                 roleRepository.save(role);
             }
         }
+
+        for (int i = 0; i < 3; i++) {
+            Business business0  = Business.builder()
+                    .city("Kiev" + i)
+                    .country("Ukraine")
+                    .name("Test"+i)
+                    .build();
+            businessRepository.save(business0);
+        }
+
+        for (int i = 0; i < 3; i++) {
+            Department department0 = Department.builder()
+                    .business(Business.builder().build())
```

Comment:
I believe the valid data should be mapped on a real data in DB, change it to mapping newly created business